### DISCUSSION TITLE
How should I parse the data obtained through useFetch?

### DISCUSSION THREAD

**Discussion by fengluo2** [nuxt/nuxt#15923] [Questions]
My server.js:
```
const express = require('express')
const app = express();
······
app.get("/get/students", (request, response) => {  //students为请求路径
    const students = [
        {id: "1", name: "zhangsan", age: 18},
        {id: "2", name: "wangwu", age: 19},
        {id: "3", name: "zhaoliu", age: 28},
    ]
    response.send(students)
})
······
```
My test components
```
······
<script lang="ts" setup>
const {data: region} = useLazyFetch('http://localhost:8080/get/students', {
  method: 'GET'
})
console.log(region.value[0].id)
</script>
······
```
He should output '1',but reports an error: Cannot read properties of null (reading '0')
How should I get data

**Comment by mcrapts** [comment]
Look at the docs: https://nuxt.com/docs/api/composables/use-lazy-fetch

You should watch `region` and act on it when it changes. Or, alternatively, you could use `await useFetch` instead of `useLazyFetch`. The latter is more common because it will render the result server-side. Server-side rendering is usually why people use Nuxt.
